packages/terra-button-group/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Changelog
 Unreleased
 ----------
 ### Changed
+
+* Inline SVG aligned for IE and wdio screenshots Updated.
+* Import for package version from package.json added to doc page
```

Comment:
This line doesn't seem to relate to the changes in the PR.

Reply:
Resolved in 2ff8a50